ui/src/app/services/error-handling.service.ts

```diff
@@ -52,35 +65,28 @@ export class ErrorHandlingService {
     this.serverBusy = false;
   }
 
-  // Don't retry API calls unless the status code is 503.
+  // don't retry API calls unless the status code is 503.
   public retryApi (observable: Observable<any>,
```

Comment:
NIT: no line wrap here (surely we're under 100 characters)

Reply:
done